taxonomies/packaging_materials.txt

```diff
@@ -112,300 +217,713 @@ fr:Plastiques mixtes
-hr:HDPE - polietilen visoke gustoće            
+hr:HDPE - polietilen visoke gustoće, polietilen visoke gustoće        
 hu:HDPE - Nagy sűrűségű polietilén, Nagy sűrűségű polietilén
-it:Polietilene ad alta densità
-nl:HDPE - Hogedichtheidpolyetheen, PE-HD
+it:HDPE - Polietilene ad alta densità, Polietilene ad alta densità
+lt:HDPE - Didelio tankio polietilenas, Didelio tankio polietilenas
+nl:HDPE - Hogedichtheidpolyetheen, Hogedichtheidpolyetheen
+nn:HDPE - høgdensitetspolyetylen, høgdensitetspolyetylen
+pl:HDPE - polietylen wysokiej gęstości, polietylen wysokiej gęstości
 pt:HDPE - Polietileno de alta densidade, Polietileno de alta densidade
+ru:HDPE - полиэтилен высокой плотности, полиэтилен высокой плотности
+uk:HDPE - Поліетилен високої щільності, Поліетилен високої щільності
 wikidata:en:Q2641430
 non_recyclable_and_non_biodegradable:en:no
+description:de:Polyethylen hoher Dichte (HDPE) wird aus Polyethylen hergestellt, das aus Erdöl gewonnen wird. HDPE wird als Rohstoff für härtere Kunststoffprodukte (Verschlüsse, Flaschen, Beutel) verwendet. HDPE kann als Rohstoff wiederverwendet werden.
+description:en:High-density polyethylene (HDPE) is made from polyethylene that is extracted from petroleum. HDPE is used as a raw material for harder plastic products (caps, bottles, bags). HDPE can be reused as a raw material.
+description:es:El polietileno de alta densidad (HDPE) está hecho de polietileno que se extrae del petróleo. El HDPE se utiliza como materia prima para productos de plástico más duro (tapas, botellas, bolsas). El HDPE se puede reutilizar como materia prima.
+description:fr:Le polyéthylène haute densité (HDPE) est fabriqué à partir de polyéthylène extrait du pétrole. Le HDPE est utilisé comme matière première pour les produits en plastique plus durs (bouchons, bouteilles, sacs). Le PEHD peut être réutilisé comme matière première.
+description:it:Il polietilene ad alta densità (HDPE) è costituito da polietilene estratto dal petrolio. L'HDPE è utilizzato come materia prima per prodotti in plastica più dura (tappi, bottiglie, sacchetti). L'HDPE può essere riutilizzato come materia prima.
+description:nl:Hogedichtheidpolyetheen (HDPE) wordt gemaakt van polyetheen dat uit aardolie wordt gewonnen. HDPE wordt gebruikt als grondstof voor hardere kunststof producten (doppen, flessen, zakken). HDPE kan opnieuw gebruikt worden als grondstof.
+description:pt:O polietileno de alta densidade (PEAD) é feito de polietileno extraído do petróleo. O HDPE é utilizado como matéria-prima para produtos plásticos mais duros (tampas, garrafas, sacolas). O HDPE pode ser reutilizado como matéria-prima.
 
 <xx:HDPE
 <xx:Bio-PE
-en:Biobased HDPE, Biobased PE-HD
-de:Biobasiertes HDPE, Biobasiertes PE-HD
-fr:PEHD biosourcé, HDPE biosourcé
-hu:Biobázisú HDPE
-pt:Bioplástico PEHD
-xx:Bio-HDPE
+en:bioHDPE - Biobased HDPE, Biobased PE-HD
+de:bioHDPE - Biobasiertes HDPE, Biobasiertes PE-HD
+fr:bioHDPE - PEHD biosourcé, HDPE biosourcé
```

Comment:
I think we should also keep the entry without bioHDPE to avoid disabling robotoff from matching "PEHD biosourcé" with this entry.

Reply:
@aleene I confirm this kind of modifications make some test fails, because we match taxonomy when we try to parse the packaging texts.

Eg. in `"1 wooden box to recycle, 6 25cl glass bottles to reuse, 3 steel lids to recycle, 1 plastic film to discard"` there is now an unrecognized material.

Do you want to